src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -0,0 +1,40 @@
+package juja.sqlcmd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
```

Comment:
Two blank lines for what purpose?

Reply:
Thanks for the comment, I should be more careful